### PR TITLE
feat: Bump REPLICA_REV to 41319eab85e65e3eb8174f37518bf81530a58831

### PR DESCRIPTION
**Pull request by DanielThurau**
Bump the replica revision for the nns and sns extensions.

### CODE REVIEW

**Comment by DanielThurau** [comment]
I'll bump the SNS version in a separate MR as its e2e tests require the nns extension to be at the version this PR is setting it to.

File: extensions/nns/build.rs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 use std::env;
 use std::path::PathBuf;
 
-const REPLICA_REV: &str = "f99495f3772d5a85d25ef5008179b49a5f12c5c2";
+const REPLICA_REV: &str = "8dd5cd00acdc4d891fa0c5a1605bcf3b68a3d707";
```
Comment (ericswanson-dfinity) [inline extensions/nns/build.rs:4]: Should this commit match the one in the changelogs?

**Comment by smallstepman** [comment]
ill take a look at the e2e errors tomorrow 🙄, silly that it happens, sorry

File: extensions/nns/build.rs (hunk 1 above)
Comment (DanielThurau) [reply extensions/nns/build.rs:4]: Yes it should. Head slap

File: extensions/nns/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased] - ReleaseDate
 
+- Same functionality as version `0.2.1`.
+- Bump the REPLICA_REV to `41319eab85e65e3eb8174f37518bf81530a58831` which installs the
```
Comment (DanielThurau) [inline extensions/nns/CHANGELOG.md:6]:
```suggestion
- Bump the REPLICA_REV to `8dd5cd00acdc4d891fa0c5a1605bcf3b68a3d707` which installs the
```

File: Cargo.toml [hunk 3]
```diff
@@ -46,6 +46,10 @@ unix-archive = ".tar.gz"
 checksum = "sha256"
 dist = true
 allow-dirty = ["ci"]
+# The preferred cargo-dist version to use in CI (Cargo.toml SemVer syntax)
+cargo-dist-version = "0.0.7"
+# The installers to generate for each app
+installers = []
```
Comment (DanielThurau) [inline Cargo.toml:52]:
```suggestion
```

**Comment by sesi200** [comment]
Closing as outdated since I just pulled in the latest replica

**Pull request closed without merging**